CHANGELOG.rst

```diff
@@ -2,6 +2,14 @@
 Changelog for package pocketsphinx
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.5.0 (2020-12-24)
```

Comment:
please revert the changes to the changelog - these get auto generated when the release is made.